metaflow/plugins/argo/test/test_run.py

```diff
@@ -0,0 +1,31 @@
+from metaflow.client.trigger_live_run import trigger_live_run  # trigger
+import time
+
+flow_name = None  # "HelloArgoFlowTwo", "FailureFlow", "MissingFlow", None
+template_name = (
+    "helloargoflowtwo"  # "helloargoflowtwo", "failureflow", "missingflow", None
+)
+run = trigger_live_run(
+    plugin_name="Argo",
+    flow_name=flow_name,
+    template_name=template_name,
+    parameters=None,
+    wait=False,
+    x=2,
+    y=55,
+    z="zzzzz top",  # testing what happens when there are additional, unused kwargs
```

Comment:
Please remove when done with prototyping. This is more a python feature rather than a behavior you are expecting.